events/management/commands/add_kasko_keywords.py

```diff
@@ -0,0 +1,197 @@
+            "en": "remote learning",
+        },
+    },
+    "helsinki:adult_education": {
+        "aggregate": False,
+        "deprecated": False,
+        "has_upcoming_events": False,
+        "data_source_id": "helsinki",
+        "publisher_id": ORGANIZATION_ID,
+        "name": {
+            "fi": "työväenopisto",
+            "sv": "arbetarinstitutet",
+            "en": "adult education",
+        },
+    },
+    "helsinki:vocational_school": {
+        "aggregate": False,
+        "deprecated": False,
+        "has_upcoming_events": True,
+        "data_source_id": "helsinki",
+        "publisher_id": ORGANIZATION_ID,
+        "name": {"fi": "ammattiopisto", "sv": "yrkesskola", "en": "vocational school"},
+    },
+    "helsinki:institution_of_higher_education": {
+        "aggregate": False,
+        "deprecated": False,
+        "has_upcoming_events": False,
+        "data_source_id": "helsinki",
+        "publisher_id": ORGANIZATION_ID,
+        "name": {
+            "fi": "korkeakoulu",
+            "sv": "högskola",
+            "en": "institution of higher education",
+        },
+    },
+    "helsinki:general_upper_secondary_school": {
+        "aggregate": False,
+        "deprecated": False,
+        "has_upcoming_events": False,
+        "data_source_id": "helsinki",
+        "publisher_id": ORGANIZATION_ID,
+        "name": {
+            "fi": "lukio",
+            "sv": "gymnasiet",
+            "en": "general upper secondary school",
+        },
+    },
+    "helsinki:secondary_schools_cross_institutional_studies": {
+        "aggregate": False,
+        "deprecated": False,
+        "has_upcoming_events": False,
+        "data_source_id": "helsinki",
+        "publisher_id": ORGANIZATION_ID,
+        "name": {
+            "fi": "toisen asteen ristiinopiskelu",
+            "sv": "andra stadiets korsstudier",
+            "en": "secondary schools cross-institutional studies",
+        },
+    },
+}
+
+
+class Command(BaseCommand):
+    help = "Adds kasko keywords."
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        self.create_keywords()
+        self.create_keyword_set()
+
+    def create_keywords(self):
+        for kw_id, kw_data in KEYWORDS.items():
+            kw_data = kw_data.copy()
+            kw_data["id"] = kw_id
+            self._create_kw_object(Keyword, kw_data)
+
+    def create_keyword_set(self):
+        for kw_set_data in KEYWORD_SET:
+            kw_set_data = kw_set_data.copy()
+            self._create_keyword_set_with_keywords(kw_set_data)
+
+    def _create_keyword_set_with_keywords(self, kw_set_data) -> KeywordSet:
+        keyword_ids = kw_set_data.pop("keywords")
+        kw_set = self._create_kw_object(KeywordSet, kw_set_data)
+
+        if kw_set and keyword_ids:
+            for kw_id in keyword_ids:
+                kw = Keyword.objects.filter(id=kw_id).first()
+                if kw:
+                    kw_set.keywords.add(kw)
```

Comment:
Not sure why we need to check if `kw_set` or `keyword_ids` is falsy 🤔 Also, I think raising an exception in case of a missing keyword is safer than just silently ignoring it. So my suggestion:

```suggestion
        for kw_id in keyword_ids:
            kw_set.add(Keyword.objects.get(id=kw_id))
```